CREATE DATABASE  IF NOT EXISTS `hematologicDB` /*!40100 DEFAULT CHARACTER SET utf8 */ /*!80016 DEFAULT ENCRYPTION='N' */;
USE `hematologicDB`;
-- MySQL dump 10.13  Distrib 8.0.22, for Win64 (x86_64)
--
-- Host: 127.0.0.1    Database: hematologicDB
-- ------------------------------------------------------
-- Server version	8.0.23

/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
/*!50503 SET NAMES utf8 */;
/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
/*!40103 SET TIME_ZONE='+00:00' */;
/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;

--
-- Table structure for table `Disease`
--

DROP TABLE IF EXISTS `Disease`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `Disease` (
  `idDisease` int unsigned NOT NULL AUTO_INCREMENT,
  `Name` longtext,
  `ICD_10` varchar(45) DEFAULT NULL,
  `Symptoms_idSymptoms` int NOT NULL,
  PRIMARY KEY (`idDisease`,`Symptoms_idSymptoms`),
  KEY `fk_Disease_Symptoms1_idx` (`Symptoms_idSymptoms`),
  CONSTRAINT `fk_Disease_Symptoms1` FOREIGN KEY (`Symptoms_idSymptoms`) REFERENCES `Symptoms` (`idSymptoms`)
) ENGINE=InnoDB AUTO_INCREMENT=15 DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Disease`
--

LOCK TABLES `Disease` WRITE;
/*!40000 ALTER TABLE `Disease` DISABLE KEYS */;
INSERT INTO `Disease` VALUES (1,'Lymphoid leukaemia',' C91',1),(2,'Myeloid leukaemia','C92',2),(3,'Monocytic leukaemia','C93',3),(4,'Iron deficiency anaemia',' D50',4),(5,'Vitamin B12 deficiency anaemia','D51',5),(6,'Folate deficiency anaemia','D52',6),(7,'Disseminated intravascular coagulation [defibrination syndrome]','D65',7),(8,'Hereditary factor VIII deficiency','D66',8),(9,'Hereditary factor IX deficiency','D67',9),(10,'Hodgkin lymphoma',' C81',10),(11,'Follicular lymphoma','C82',11),(12,'Non-follicular lymphoma','C83',12),(13,'Multiple myeloma',' C90.0',13),(14,'Thrombocytopenia',' D69.6',14);
/*!40000 ALTER TABLE `Disease` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Genes`
--

DROP TABLE IF EXISTS `Genes`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `Genes` (
  `idGenes` int NOT NULL,
  `Symbol` varchar(45) DEFAULT NULL,
  `Full_name` longtext,
  `Type` varchar(45) DEFAULT NULL,
  `Link_sequence` longtext,
  PRIMARY KEY (`idGenes`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Genes`
--

LOCK TABLES `Genes` WRITE;
/*!40000 ALTER TABLE `Genes` DISABLE KEYS */;
INSERT INTO `Genes` VALUES (1,'KMT2C','lysine methyltransferase 2C','gene with protein product','https://www.ncbi.nlm.nih.gov/nuccore/NC_000007.14?report=fasta&from=152134925&to=152436642&strand=true'),(2,'KMT2A','lysine methyltransferase 2A','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000011.10?report=fasta&from=118436492&to=118526832'),(3,'BCR','BCR activator of RhoGEF and GTPase','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000022.11?report=fasta&from=23180509&to=23318037'),(4,'RUNX1','RUNX family transcription factor 1','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000021.9?report=fasta&from=34787801&to=35049334&strand=true'),(5,'CBFA2T2','CBFA2/RUNX1 partner transcriptional co-repressor 2','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000020.11?report=fasta&from=33490070&to=33650031'),(6,'KAT6A','lysine acetyltransferase 6A','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000008.11?report=fasta&from=41929479&to=42051988&strand=true'),(7,'BRD1','bromodomain containing 1','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000022.11?report=fasta&from=49773278&to=49827873&strand=true'),(8,'TMPRSS6','transmembrane serine protease 6','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000022.11?report=fasta&from=37065436&to=37110625&strand=true'),(9,'MT-CO1','mitochondrially encoded cytochrome c oxidase I','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_012920.1?report=fasta&from=5904&to=7445'),(10,'GLRX5','glutaredoxin 5','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000014.9?report=fasta&from=95535050&to=95544714'),(11,'CUBN','cubilin','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000010.11?report=fasta&from=16823966&to=17130492&strand=true'),(12,'MTR','5-methyltetrahydrofolate-homocysteine methyltransferase','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000001.11?report=fasta&from=236794304&to=236903981'),(13,'CBLIF','cobalamin binding intrinsic factor','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000011.10?report=fasta&from=59829268&to=59845501&strand=true'),(14,'SLC46A1','solute carrier family 46 member 1','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000017.11?report=fasta&from=28394642&to=28406630&strand=true'),(15,'DHFR','dihydrofolate reductase','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000005.10?report=fasta&from=80626226&to=80654983&strand=true'),(16,'MTRR','5-methyltetrahydrofolate-homocysteine methyltransferase reductase','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000005.10?report=fasta&from=7851186&to=7901124'),(17,'HFE','homeostatic iron regulator','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000006.12?report=fasta&from=26087347&to=26098343'),(18,'ADAMTS13','ADAM metallopeptidase with thrombospondin type 1 motif 13','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000009.12?report=fasta&from=133414337&to=133459403'),(19,'VWF','von Willebrand factor','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000012.12?report=fasta&from=5948877&to=6124670&strand=true'),(20,'GPI','glucose-6-phosphate isomerase','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000019.10?report=fasta&from=34353330&to=34402413'),(21,'BCL6','BCL6 transcription repressor','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000003.12?report=fasta&from=187721377&to=187745472&strand=true'),(22,'BCL10','BCL10 immune signaling adaptor','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000001.11?report=fasta&from=85265776&to=85276632&strand=true'),(23,'ALK','ALK receptor tyrosine kinase','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000002.12?report=fasta&from=29190992&to=29921589&strand=true'),(24,'KDSR','3-ketodihydrosphingosine reductase','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000018.10?report=fasta&from=63327726&to=63367234&strand=true'),(25,'EZH2','enhancer of zeste 2 polycomb repressive complex 2 subunit','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000007.14?report=fasta&from=148807374&to=148884344&strand=true'),(26,'NSD2','nuclear receptor binding SET domain protein 2','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000004.12?report=fasta&from=1871393&to=1982207'),(27,'IRF4','interferon regulatory factor 4','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000006.12?report=fasta&from=391752&to=411443'),(28,'MMTAG2','chromosome 1 open reading frame 35','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000001.11?report=fasta&from=228100731&to=228103325&strand=true'),(29,'GATA1','GATA binding protein 1','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000023.11?report=fasta&from=48786590&to=48794311'),(30,'WAS','WASP actin nucleation promoting factor','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000023.11?report=fasta&from=48683753&to=48691427'),(31,'ITGB3','integrin subunit beta 3','protein coding','https://www.ncbi.nlm.nih.gov/nuccore/NC_000017.11?report=fasta&from=47253827&to=47313743');
/*!40000 ALTER TABLE `Genes` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Protein`
--

DROP TABLE IF EXISTS `Protein`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `Protein` (
  `idProtein` int NOT NULL,
  `Name` longtext,
  `Molec_function` longtext,
  `Bio_proces` longtext,
  `Sequence` longtext,
  `Proteome` varchar(45) DEFAULT NULL,
  `Genes_idGenes` int NOT NULL,
  `Disease_idDisease` int unsigned NOT NULL,
  `Disease_Symptoms_idSymptoms` int NOT NULL,
  PRIMARY KEY (`idProtein`,`Genes_idGenes`,`Disease_idDisease`,`Disease_Symptoms_idSymptoms`),
  KEY `fk_Protein_Genes1_idx` (`Genes_idGenes`),
  KEY `fk_Protein_Disease1_idx` (`Disease_idDisease`,`Disease_Symptoms_idSymptoms`),
  CONSTRAINT `fk_Protein_Disease1` FOREIGN KEY (`Disease_idDisease`, `Disease_Symptoms_idSymptoms`) REFERENCES `Disease` (`idDisease`, `Symptoms_idSymptoms`),
  CONSTRAINT `fk_Protein_Genes1` FOREIGN KEY (`Genes_idGenes`) REFERENCES `Genes` (`idGenes`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Protein`
--

LOCK TABLES `Protein` WRITE;
/*!40000 ALTER TABLE `Protein` DISABLE KEYS */;
INSERT INTO `Protein` VALUES (1,'Histone-lysine N-methyltransferase 2C','DNA binding,histone binding,histone methyltransferase activity,histone methyltransferase activity, metal ion binding,RNA binding,transcription coactivator activity,transferase activity, transferring acyl groups','histone H3-K4 monomethylation,positive regulation of transcription by RNA polymerase II,regulation of megakaryocyte differentiation','https://www.uniprot.org/uniprot/Q8NEZ4.fasta','Chromosome 7',1,1,1),(2,'Histone-lysine N-methyltransferase 2A','histone methyltransferase activity (H3-K4 specific), identical protein binding,lysine-acetylated histone binding,minor groove of adenine-thymine-rich DNA binding, protein homodimerization activity, unmethylated CpG binding, zinc ion binding  ','apoptotic process Source, circadian regulation of gene expression Source, embryonic hemopoiesis Source, histone H3-K4 dimethylation Source, histone H3-K4 methylation Source, histone H3-K4 monomethylation Source, histone H3-K4 trimethylation Source, histone H4-K16 acetylation Source, negative regulation of DNA methylation Source, positive regulation of histone H3-K4 methylation Source, positive regulation of transcription, DNA-templated Source, positive regulation of transcription by RNA polymerase II Source, positive regulation of transporter activity Source, protein-containing complex assembly Source: UniProtKB, regulation of hematopoietic stem cell differentiation Source, regulation of histone H3-K14 acetylation Source, regulation of histone H3-K9 acetylation Source, regulation of megakaryocyte differentiation Source','https://www.uniprot.org/uniprot/Q03164.fasta','Chromosome 11',2,1,1),(3,'Breakpoint cluster region protein','ATP binding Source, enzyme binding Source, GTPase activator activity Source, guanyl-nucleotide exchange factor activity Source, protein serine/threonine kinase activity Source, protein serine kinase activity Source, protein threonine kinase activity Source','activation of GTPase activity Source, focal adhesion assembly Source, keratinocyte differentiation Source, modulation of chemical synaptic transmission Source, platelet-derived growth factor receptor signaling pathway Source, protein autophosphorylation Source, protein phosphorylation Source, regulation of Rho protein signal transduction Source ,regulation of small GTPase mediated signal transduction Source, signal transduction Source, small GTPase mediated signal transduction','https://www.uniprot.org/uniprot/P11274.fasta','Chromosome 22',3,1,1),(4,'Histone-lysine N-methyltransferase 2A','histone methyltransferase activity (H3-K4 specific), identical protein binding,lysine-acetylated histone binding,minor groove of adenine-thymine-rich DNA binding, protein homodimerization activity, unmethylated CpG binding, zinc ion binding  ','apoptotic process Source, circadian regulation of gene expression Source, embryonic hemopoiesis Source, histone H3-K4 dimethylation Source, histone H3-K4 methylation Source, histone H3-K4 monomethylation Source, histone H3-K4 trimethylation Source, histone H4-K16 acetylation Source, negative regulation of DNA methylation Source, positive regulation of histone H3-K4 methylation Source, positive regulation of transcription, DNA-templated Source, positive regulation of transcription by RNA polymerase II Source, positive regulation of transporter activity Source, protein-containing complex assembly Source: UniProtKB, regulation of hematopoietic stem cell differentiation Source, regulation of histone H3-K14 acetylation Source, regulation of histone H3-K9 acetylation Source, regulation of megakaryocyte differentiation Source','https://www.uniprot.org/uniprot/Q03164.fasta','Chromosome 11',2,2,2),(5,'Runt-related transcription factor 1','ATP binding Source,calcium ion binding Source, DNA binding Source, DNA-binding transcription activator activity, RNA polymerase II-specific Source, DNA-binding transcription factor activity Source, DNA-binding transcription factor activity, RNA polymerase II-specific Source, prot,in heterodimerization activity Source, protein homodimerization activity Source, RNA polymerase II cis-regulatory region sequence-specific DNA binding Source, RNA polymerase II transcription regulatory region sequence-specific DNA binding Sourcetra, scription factor binding Source, transcription regulatory region sequence-specific DNA binding','cardiac muscle tissue regeneration Source, chondrocyte differentiation Source, hematopoietic stem cell proliferation Source, hemopoiesis Source, myeloid cell differentiation Source, myeloid leukocyte differentiation Source, negative regulation of CD4-positive, alpha-beta T cell differentiation Source, negative regulation of gene expression Source, negative regulation of granulocyte differentiation Source, negative regulation of transcription by RNA polymerase II Source, neuron differentiation Source, ossification Source, peripheral nervous system neuron development Source, positive regulation of angiogenesis Source, positive regulation of CD8-positive, alpha-beta T cell differentiation Source, positive regulation of collagen biosynthetic process Source, positive regulation of extracellular matrix organization Source, positive regulation of granulocyte differentiation Source, positive regulation of interleukin-2 production Source, positive regulation of transcription, DNA-templated Source, positive regulation of transcription by RNA polymerase II Source, regulation of B cell receptor signaling pathway Source, regulation of bicellular tight junction assembly Source, regulation of cardiac muscle cell proliferation Source, regulation of cell differentiation Source, regulation of connective tissue replacement Source, regulation of cytokine-mediated signaling pathway Source, regulation of hematopoietic stem cell differentiation Source, regulation of intracellular estrogen receptor signaling pathway Source, regulation of keratinocyte differentiation Source, regulation of megakaryocyte differentiation Source, regulation of myeloid cell differentiation Source, regulation of plasminogen activation Source, regulation of regulatory T cell differentiation Source,regulation of transcription by RNA polymerase II Source,regulation of Wnt signaling pathway','https://www.uniprot.org/uniprot/Q01196.fasta','Chromosome 21',4,2,2),(6,'Protein CBFA2T2','metal ion binding, transcription corepressor activity','intestinal epithelial cell differentiation Source, negative regulation of neuron projection development Source, negative regulation of Notch signaling pathway Source, negative regulation of transcription, DNA-templated Source, negative regulation of transcription by RNA polymerase II Source, positive regulation of neuron projection development Source, transcription, DNA-templated','https://www.uniprot.org/uniprot/O43439.fasta','Chromosome 20',5,2,2),(7,'Histone acetyltransferase KAT6A','acetyltransferase activity Source, DNA binding Source, histone acetyltransferase activity Source, histone binding Source, transcription coactivator activity Source, transcription coregulator activity Source, transcription factor binding Source, zinc ion binding','cellular senescence Source, DNA packaging Source, histone acetylation Source, histone H3 acetylation Source, myeloid cell differentiation Source, negative regulation of transcription, DNA-templated Source, nucleosome assembly Source, positive regulation of transcription, DNA-templated Source, positive regulation of transcription by RNA polymerase II Source, protein acetylation Source, regulation of signal transduction by p53 class mediator','https://www.uniprot.org/uniprot/Q92794.fasta','Chromosome 8',6,3,3),(7,'Bromodomain-containing protein 1','histone binding, metal ion binding','erythrocyte maturation Source, histone H3 acetylation Source, histone H3-K14 acetylation Source, histone H3-K23 acetylation Source, positive regulation of erythrocyte differentiation Source, response to electrical stimulus Source, response to immobilization stress','https://www.uniprot.org/uniprot/O95696.fasta','Chromosome 22',7,3,3),(8,'Runt-related transcription factor 1','ATP binding Source,calcium ion binding Source, DNA binding Source, DNA-binding transcription activator activity, RNA polymerase II-specific Source, DNA-binding transcription factor activity Source, DNA-binding transcription factor activity, RNA polymerase II-specific Source, prot,in heterodimerization activity Source, protein homodimerization activity Source, RNA polymerase II cis-regulatory region sequence-specific DNA binding Source, RNA polymerase II transcription regulatory region sequence-specific DNA binding Sourcetra, scription factor binding Source, transcription regulatory region sequence-specific DNA binding','cardiac muscle tissue regeneration Source, chondrocyte differentiation Source, hematopoietic stem cell proliferation Source, hemopoiesis Source, myeloid cell differentiation Source, myeloid leukocyte differentiation Source, negative regulation of CD4-positive, alpha-beta T cell differentiation Source, negative regulation of gene expression Source, negative regulation of granulocyte differentiation Source, negative regulation of transcription by RNA polymerase II Source, neuron differentiation Source, ossification Source, peripheral nervous system neuron development Source, positive regulation of angiogenesis Source, positive regulation of CD8-positive, alpha-beta T cell differentiation Source, positive regulation of collagen biosynthetic process Source, positive regulation of extracellular matrix organization Source, positive regulation of granulocyte differentiation Source, positive regulation of interleukin-2 production Source, positive regulation of transcription, DNA-templated Source, positive regulation of transcription by RNA polymerase II Source, regulation of B cell receptor signaling pathway Source, regulation of bicellular tight junction assembly Source, regulation of cardiac muscle cell proliferation Source, regulation of cell differentiation Source, regulation of connective tissue replacement Source, regulation of cytokine-mediated signaling pathway Source, regulation of hematopoietic stem cell differentiation Source, regulation of intracellular estrogen receptor signaling pathway Source, regulation of keratinocyte differentiation Source, regulation of megakaryocyte differentiation Source, regulation of myeloid cell differentiation Source, regulation of plasminogen activation Source, regulation of regulatory T cell differentiation Source,regulation of transcription by RNA polymerase II Source,regulation of Wnt signaling pathway','https://www.uniprot.org/uniprot/Q01196.fasta','Chromosome 21',4,3,3),(8,'Transmembrane protease serine 6','serine-type endopeptidase activity','cellular iron ion homeostasis Source, collagen catabolic process Source, extracellular matrix disassembly Source, iron ion homeostasis Source, membrane protein proteolysis Source, negative regulation of BMP signaling pathway Source, negative regulation of transcription, DNA-templated Source, negative regulation of transcription by RNA polymerase II Source, positive regulation of transcription by RNA polymerase II Source, self proteolysis','https://www.uniprot.org/uniprot/Q8IU80.fasta','Chromosome 22',8,4,4),(9,'Cytochrome c oxidase subunit 1','cytochrome-c oxidase activity, heme binding, metal ion binding','aerobic respiration Source, aging Source, cerebellum development Source, electron transport coupled proton transport Source, mitochondrial electron transport, cytochrome c to oxygen Source, respiratory electron transport chain Source, response to copper ion Source, response to electrical stimulus Source, response to oxidative stress','https://www.uniprot.org/uniprot/P00395.fasta','Mitochondrion',9,4,4),(10,'Glutaredoxin-related protein 5, mitochondrial','2 iron, 2 sulfur cluster binding Source, metal ion binding Source, protein disulfide oxidoreductase activity','[2Fe-2S] cluster assembly Source, hemopoiesis Source, iron-sulfur cluster assembly Source, protein maturation by [2Fe-2S] cluster transfer Source, protein maturation by [4Fe-4S] cluster transfer','https://www.uniprot.org/uniprot/Q86SX6.fasta','Chromosome 14',10,4,4),(11,'Cubilin','calcium ion binding Source, cargo receptor activity Source, cobalamin binding Source, protein homodimerization activity Source, signaling receptor activity','cholesterol metabolic process Source,cobalamin metabolic process Source, cobalamin transport Source, high-density lipoprotein particle clearance Source, lipoprotein transport Source, receptor-mediated endocytosis Source, response to bacterium Source, tissue homeostasis Source, vitamin D metabolic process','https://www.uniprot.org/uniprot/O60494.fasta','Chromosome 10',11,5,5),(12,'Methionine synthase','cobalamin binding Source, methionine synthase activity Source, zinc ion binding','axon regeneration Source,cellular response to nitric oxide Source ,cobalamin metabolic process Source, methionine biosynthetic process Source, methylation Source, nervous system development Source, pteridine-containing compound metabolic process Source, response to axon injury Source, sulfur amino acid metabolic process','https://www.uniprot.org/uniprot/Q99707.fasta','Chromosome 1',12,5,5),(13,'Cobalamin binding intrinsic factor','cobalamin binding','cobalamin metabolic process Source, cobalamin transport Source, cobalt ion transport','https://www.uniprot.org/uniprot/P27352.fasta','Chromosome 11',13,5,5),(14,'Proton-coupled folate transporter','folic acid binding, folic acid transmembrane transporter activity','cellular iron ion homeostasis, folate import across plasma membrane','https://www.uniprot.org/uniprot/Q96NT5.fasta','Chromosome 17',14,6,6),(15,'Dihydrofolate reductase','dihydrofolate reductase activity,drug binding','axon regeneration, dihydrofolate metabolic process','https://www.uniprot.org/uniprot/P00374.fasta','Chromosome 5',15,6,6),(16,'Methionine synthase reductase','[methionine synthase] reductase activity,aquacobalamin reductase (NADPH) activity','cobalamin metabolic process,DNA methylation','https://www.uniprot.org/uniprot/Q9UBK8.fasta','Chromosome 5',16,6,6),(19,'Hereditary hemochromatosis protein','beta-2-microglobulin binding, co-receptor binding','acute-phase response, BMP signaling pathway','https://www.uniprot.org/uniprot/Q30201.fasta','Chromosome 6',17,8,8),(20,'A disintegrin and metalloproteinase with thrombospondin motifs 13','calcium ion binding,integrin binding','cell-matrix adhesion, cellular response to interferon-gamma','https://www.uniprot.org/uniprot/Q76LX8.fasta','Chromosome 9',18,8,8),(21,'von Willebrand factor','chaperone binding, collagen binding','blood coagulation, blood coagulation, intrinsic pathway','https://www.uniprot.org/uniprot/P04275.fasta','Chromosome 12',19,8,8),(22,'Hereditary hemochromatosis protein','beta-2-microglobulin binding, co-receptor binding','acute-phase response, BMP signaling pathway','https://www.uniprot.org/uniprot/Q30201.fasta','Chromosome 6',17,9,9),(23,'Glucose-6-phosphate isomerase','cytokine activity, glucose-6-phosphate isomerase activity','canonical glycolysis, carbohydrate metabolic process','https://www.uniprot.org/uniprot/P06744.fasta','Chromosome 19',20,9,9),(24,'A disintegrin and metalloproteinase with thrombospondin motifs 13','calcium ion binding,integrin binding','cell-matrix adhesion, cellular response to interferon-gamma','https://www.uniprot.org/uniprot/Q76LX8.fasta','Chromosome 9',18,9,9),(25,'B-cell lymphoma 6 protein','chromatin binding, chromatin DNA binding','actin cytoskeleton organization, B cell differentiation','https://www.uniprot.org/uniprot/P41182.fasta','Chromosome 3',21,10,10),(26,'B-cell lymphoma/leukemia 10','CARD domain binding, enzyme binding','adaptive immune response, B cell apoptotic process','https://www.uniprot.org/uniprot/O95999.fasta','Chromosome 1',22,10,10),(27,'B-cell lymphoma 6 protein','chromatin binding, chromatin DNA binding','actin cytoskeleton organization, B cell differentiation','https://www.uniprot.org/uniprot/P41182.fasta','Chromosome 3',21,11,11),(27,'ALK tyrosine kinase receptor','ATP binding, identical protein binding','activation of MAPK activity, adult behavior','https://www.uniprot.org/uniprot/Q9UM73.fasta','Chromosome 2',23,10,10),(28,'B-cell lymphoma/leukemia 10','CARD domain binding, enzyme binding','adaptive immune response, B cell apoptotic process','https://www.uniprot.org/uniprot/O95999.fasta','Chromosome 1',22,11,11),(28,'3-ketodihydrosphingosine reductase','3-dehydrosphinganine reductase activity','3-keto-sphinganine metabolic process, sphingolipid biosynthetic process','https://www.uniprot.org/uniprot/Q06136.fasta','Chromosome 18',24,11,11),(29,'Histone-lysine N-methyltransferase EZH2','chromatin binding, chromatin DNA binding, histone-lysine N-methyltransferase activity','B cell differentiation, cardiac muscle hypertrophy in response to stress','https://www.uniprot.org/uniprot/Q15910.fasta','Chromosome 7',25,12,12),(30,'B-cell lymphoma 6 protein','chromatin binding, chromatin DNA binding','actin cytoskeleton organization, B cell differentiation','https://www.uniprot.org/uniprot/P41182.fasta','Chromosome 3',21,12,12),(31,'B-cell lymphoma/leukemia 10','CARD domain binding, enzyme binding','adaptive immune response, B cell apoptotic process','https://www.uniprot.org/uniprot/O95999.fasta','Chromosome 1',22,12,12),(32,'Histone-lysine N-methyltransferase NSD2','chromatin binding, histone methyltransferase activity (H3-K36 specific)','atrial septum primum morphogenesis, atrial septum secundum morphogenesis','https://www.uniprot.org/uniprot/O96028.fasta','Chromosome 4',26,13,13),(33,'Interferon regulatory factor 4','DNA-binding transcription activator activity, RNA polymerase II-specific, DNA-binding transcription factor activity','cytokine-mediated signaling pathway, defense response to protozoan','https://www.uniprot.org/uniprot/Q15306.fasta','Chromosome 6',27,13,13),(34,'Multiple myeloma tumor-associated protein 2','RNA binding','neutrophil degranulation','https://www.uniprot.org/uniprot/Q9BU76.fasta','Chromosome 1',28,13,13),(35,'Erythroid transcription factor','C2H2 zinc finger domain binding, chromatin DNA binding','basophil differentiation,blood coagulation','https://www.uniprot.org/uniprot/P15976.fasta','Chromosome X',29,14,14),(36,'Wiskott-Aldrich syndrome protein','actin binding, GTPase regulator activity','actin filament-based movement, actin filament polymerization','https://www.uniprot.org/uniprot/P42768.fasta','Chromosome X',30,14,14),(37,'Integrin beta-3','cell adhesion molecule binding, coreceptor activity','activation of protein kinase activity, angiogenesis involved in wound healing','https://www.uniprot.org/uniprot/P05106.fasta','Chromosome 17',31,14,14);
/*!40000 ALTER TABLE `Protein` ENABLE KEYS */;
UNLOCK TABLES;

--
-- Table structure for table `Symptoms`
--

DROP TABLE IF EXISTS `Symptoms`;
/*!40101 SET @saved_cs_client     = @@character_set_client */;
/*!50503 SET character_set_client = utf8mb4 */;
CREATE TABLE `Symptoms` (
  `idSymptoms` int NOT NULL,
  `Pallor` tinyint(1) DEFAULT NULL,
  `Fatigne` tinyint(1) DEFAULT NULL,
  `Weakness` tinyint(1) DEFAULT NULL,
  `Fever` tinyint(1) DEFAULT NULL,
  `Dizziness` tinyint(1) DEFAULT NULL,
  `Shortness_breath` tinyint(1) DEFAULT NULL,
  `Otros` longtext,
  PRIMARY KEY (`idSymptoms`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8;
/*!40101 SET character_set_client = @saved_cs_client */;

--
-- Dumping data for table `Symptoms`
--

LOCK TABLES `Symptoms` WRITE;
/*!40000 ALTER TABLE `Symptoms` DISABLE KEYS */;
INSERT INTO `Symptoms` VALUES (1,1,1,1,1,0,0,' Weight loss, Headaches'),(2,1,1,1,1,0,0,' Weight loss, Headaches'),(3,1,1,1,1,1,0,' Weight loss, Headaches'),(4,1,0,0,0,1,1,' Bounding pulse, Flow murmurs, Cardiac ventricular hypertrophy'),(5,1,0,0,0,1,1,' Bounding pulse, Flow murmurs, Cardiac ventricular hypertrophy'),(6,1,0,0,0,1,1,' Bounding pulse, Flow murmurs, Cardiac ventricular hypertrophy'),(7,0,0,1,0,1,0,'heavy external bleeding, blood in the urine or stool, double vision, severe head or neck pain, repeated vomiting, difficulty walking, convulsions, or seizures'),(8,0,0,1,0,1,0,'heavy external bleeding, blood in the urine or stool, double vision, severe head or neck pain, repeated vomiting, difficulty walking, convulsions, or seizures'),(9,0,0,1,0,1,0,'heavy external bleeding, blood in the urine or stool, double vision, severe head or neck pain, repeated vomiting, difficulty walking, convulsions, or seizures'),(10,1,1,1,0,0,1,'Night sweats,Weight loss,Itching'),(11,1,1,1,0,0,1,'Night sweats,Weight loss,Itching'),(12,1,1,1,0,0,1,'Night sweats,Weight loss,Itching'),(13,1,1,1,0,0,0,'Bone pain'),(14,0,0,0,0,0,0,'Null');
/*!40000 ALTER TABLE `Symptoms` ENABLE KEYS */;
UNLOCK TABLES;
/*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;

/*!40101 SET SQL_MODE=@OLD_SQL_MODE */;
/*!40014 SET FOREIGN_KEY_CHECKS=@OLD_FOREIGN_KEY_CHECKS */;
/*!40014 SET UNIQUE_CHECKS=@OLD_UNIQUE_CHECKS */;
/*!40101 SET CHARACTER_SET_CLIENT=@OLD_CHARACTER_SET_CLIENT */;
/*!40101 SET CHARACTER_SET_RESULTS=@OLD_CHARACTER_SET_RESULTS */;
/*!40101 SET COLLATION_CONNECTION=@OLD_COLLATION_CONNECTION */;
/*!40111 SET SQL_NOTES=@OLD_SQL_NOTES */;

-- Dump completed on 2021-04-09 15:49:28
